include/gsl/span

```diff
@@ -512,23 +518,25 @@ public:
 
     constexpr reference front() const noexcept
     {
-        Expects(size() > 0);
+        Expects(size() != dynamic_extent && size() > 0);
         return data()[0];
     }
 
     constexpr reference back() const noexcept
     {
-        Expects(size() > 0);
+        Expects(size() != dynamic_extent && size() > 0);
         return data()[size() - 1];
     }
-    
-    // at and operator() are deprecated to align to the public member functions of std::span
-    [[deprecated("Use operator[]")]]
-    constexpr reference at(index_type idx) const noexcept{ return this->operator[](idx); }
-    [[deprecated("Use operator[]")]]
-    constexpr reference operator()(index_type idx) const noexcept{ return this->operator[](idx); }
-
 
+    // at and operator() are deprecated to align to the public member functions of std::span
+    [[deprecated("Use operator[]")]] constexpr reference at(index_type idx) const noexcept
```

Comment:
If you're already forcing this big breaking change on users to change signedness, perhaps removing at() at this time would also be a good idea?

Reply:
Yeah, this is as good of a time as any to remove at().